<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buttonClass</class>
 <widget class="QMainWindow" name="buttonClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>button</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>upload notes</string>
      </property>
      <widget class="QPushButton" name="upload">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>upload</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>title</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_title</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>date</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_date</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_title">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_date">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="submit_info">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>lineEdit_date</tabstop>
  <tabstop>upload</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <signal>signal1()</signal>
  <slot>on_click()</slot>
 </slots>
</ui>
